cobbler/utils.py

```diff
@@ -225,8 +226,13 @@ def is_supervisord() -> bool:
 
     This method currently checks if the path ``/usr/bin/supervisord`` exists.
     """
-    return os.path.exists("/usr/bin/supervisord")
+    server = ServerProxy('http://localhost:9001/RPC2')
 
+    try:
+      server.supervisor.getState()
+    except OSError as e:
+        return False
+    return True
```

Comment:
Same as the other suggestion:

```suggestion
    with ServerProxy('http://localhost:9001/RPC2') as server:
        try:
            server.supervisor.getState()
        except OSError as e:
            return False
        return True
```